.flake8

```diff
@@ -0,0 +1,3 @@
+[flake8]
+max-line-length=127
+exclude=training/data_preparation, training/training_code
```

Comment:
I think it's better to keep this as part of the actual command

Reply:
`flake8 . --max-line-length=127 --exclude=training/data_preparation, training/training_code`

Reply:
Different models might have different `excludes` and in the future maybe different `exceptions` too. Might be better to not clutter the `.travis.yml` file so that it can be standard across repos?